public/img/icons/unicons/ai.svg

```diff
@@ -1,27 +1,5 @@
-<svg viewBox="0 0 10 10" xmlns="http://www.w3.org/2000/svg">
-<g id="Group 204">
-<path id="Vector" d="M7.5 5C5.56722 5 4.00015 7.0148 4.00015 9.5C4.00015 7.0148 2.43309 5 0.5 5C2.43309 5 4.00015 2.9855 4.00015 0.5C4.00015 2.9855 5.56722 5 7.5 5Z" fill="url(#paint0_linear_344_481)"/>
-<path id="Vector_2" d="M9.5 2.5C8.39555 2.5 7.50009 3.39547 7.50009 4.5C7.50009 3.39547 6.60462 2.5 5.5 2.5C6.60462 2.5 7.50009 1.60467 7.50009 0.5C7.50009 1.60467 8.39555 2.5 9.5 2.5Z" fill="url(#paint1_linear_344_481)"/>
-<path id="Vector_3" d="M9.5 7C8.67166 7 8.00007 7.6716 8.00007 8.5C8.00007 7.6716 7.32847 7 6.5 7C7.32847 7 8.00007 6.3285 8.00007 5.5C8.00007 6.3285 8.67166 7 9.5 7Z" fill="url(#paint2_linear_344_481)"/>
-</g>
-<defs>
-<linearGradient id="paint0_linear_344_481" x1="0.5" y1="0.5" x2="20.6236" y2="0.33439" gradientUnits="userSpaceOnUse">
-<stop stop-color="#6E9FFF" stop-opacity="0"/>
-<stop offset="0.164022" stop-color="#6E9FFF"/>
-<stop offset="0.197917" stop-color="#6E9FFF" stop-opacity="0.8075"/>
-<stop offset="0.426692" stop-color="#6E9FFF" stop-opacity="0"/>
-</linearGradient>
-<linearGradient id="paint1_linear_344_481" x1="5.5" y1="0.5" x2="16.9987" y2="0.378333" gradientUnits="userSpaceOnUse">
-<stop stop-color="#6E9FFF" stop-opacity="0"/>
-<stop offset="0.164022" stop-color="#6E9FFF"/>
-<stop offset="0.197917" stop-color="#6E9FFF" stop-opacity="0.8075"/>
-<stop offset="0.426692" stop-color="#6E9FFF" stop-opacity="0"/>
-</linearGradient>
-<linearGradient id="paint2_linear_344_481" x1="6.5" y1="5.5" x2="15.124" y2="5.40875" gradientUnits="userSpaceOnUse">
-<stop stop-color="#6E9FFF" stop-opacity="0"/>
-<stop offset="0.164022" stop-color="#6E9FFF"/>
-<stop offset="0.197917" stop-color="#6E9FFF" stop-opacity="0.8075"/>
-<stop offset="0.426692" stop-color="#6E9FFF" stop-opacity="0"/>
-</linearGradient>
-</defs>
+<svg viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M37.3333 24C27.0252 24 18.6675 34.7456 18.6675 48C18.6675 34.7456 10.3098 24 0 24C10.3098 24 18.6675 13.256 18.6675 0C18.6675 13.256 27.0252 24 37.3333 24Z" fill="currentColor" fill-opacity="0.65"/>
```

Comment:
Should the shapes still be partly transparent?

Reply:
good question - this is the icon i was given, so i guess not? 🤔 

cc @staton-hyse11

Reply:
It would be good to unify with the rest of our icons and have it solid - no transparencies